test/jest/__tests__/invoice/invoice.spec.ts

```diff
@@ -0,0 +1,76 @@
+import {InvoiceInputFields} from 'components/invoice/interfaces/InvoiceInputFields'
+import {describe, expect, it} from '@jest/globals'
+import invoice from 'components/invoice/invoice'
+
+const defaultInput:InvoiceInputFields = {
+  amount:10000,
+  amountType:'net',
```

Comment:
Please use `constants.AMOUNT_TYPES.NET` instead of the string. Generally, I use constants instead of strings, numbers etc in cases like this.